docs/source/concepts/cl_columns.rst

```diff
@@ -0,0 +1,57 @@
+.. _concepts_cl_columns:
+
+Cleanlab Columns
+****************
+
+Introduction
+============
+
+Cleanlab Studio will run a number of analyses on your dataset to help identify potential data issues. Currently we look for the following issues in datasets:
+- Label issues
+- Outliers
+New analyses are being continuously added to Cleanlab Studio. The raw outputs of each analysis will be available to download as Cleanlab columns by calling ``studio.download_cleanlab_columns()``.
+
+Most analyses produce two columns, one being a numeric score between 0 and 1 indicating the quality of a sample for that analysis (lower score means lower quality), and the other being a boolean, which indicates whether the sample is likely to have an issue or not. The different analyses are run separately, and hence they should be treated independently. For example, samples with label issues are not necessarily outliers, and vice versa, and it is possible to have a sample with label issue marked as ``False``, but outlier marked as ``True``. Furthermore, it is possible to have samples with label issue marked as ``True`` **and** outlier marked as ``True``: these are the raw results from the analyses, and you may interpret them as appropriate (e.g., if a data point is an outlier, it may be irrelevant that it's also likely a label issue).
+
+The Cleanlab columns are listed below in the table. Detailed explanations of each column are provided in the following sections.
+
+.. list-table:: 
+   :widths: 50 25
+   :header-rows: 1
+
+   * - Cleanlab Column Name
+     - Value Type
+   * - :ref:`label_issue_score <label_issue_score>`
+     - Float
+   * - :ref:`is_label_issue <is_label_issue>`
+     - Boolean
+   * - :ref:`suggested_label <suggested_label>`
+     - String
+   * - :ref:`is_outlier <is_outlier>`
+     - Boolean
+
+Label Issues
+============
+
+.. _label_issue_score:
+``label_issue_score``
```

Comment:
should this be `label_quality_score` ? i'd expect a higher `label_issue_score` to be more likely to have a label issue